scripts/install-ubuntu.sh

```
@@ -0,0 +1,320 @@
+    echo -e "${COLOR_YELLOW}Note: Your browser will show a security warning for self-signed${COLOR_RESET}"
+    echo -e "${COLOR_YELLOW}      certificates. For production, re-run with an email for Let's Encrypt.${COLOR_RESET}"
+else
+    PROTOCOL="http"
+    FINAL_URL="http://${APACHE_SERVER_NAME}/"
+fi
+
+systemctl restart apache2
+
+# Configure allowed hostnames
+if [ -f "${OSPOS_DIR}/.env" ]; then
+    sed -i "s/app\.allowedHostnames\.0 = 'localhost'/app.allowedHostnames.0 = '${APACHE_SERVER_NAME}'/" ${OSPOS_DIR}/.env
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)